### DISCUSSION TITLE
UNABLE TO UPLOAD A LARGE COURSE

### DISCUSSION THREAD

**Discussion by riesaau** [chamilo/chamilo-lms#5872] [General]
**Describe**
A 43 MB size scorm zip will upload fine to a Chamilo course, but not a 168mb file.  I am on Chamilo 1.11.26 version at present

**To Reproduce**
1. Go to 'Learning Path
2. Click on import scorm button
3. The upload consistently stops at 17%
4. I can take the videos out to make the course smaller and then FTP them back in after i upload then course, so that is my workaround
5. I have had this issue, I think, with every upgrade of Chamilo and usually my web host sorts it out by adjusting php and their web hosting settings, but this time they can't.
My webhost asked me to send you the screen caps below...

**Expected behavior**
Course is supposed to upload without stalling, i presume, as long as it is smaller than the post_max_size?

**Screenshots**
![chrome_vcElzMd07d](https://github.com/user-attachments/assets/da0804ab-a8e8-480a-920d-c50a96749cf7)
![HwMdcUI_htCjAAMRHu](https://github.com/user-attachments/assets/47cc65ed-f6c5-4660-9417-2962bc049305)

**Desktop (please complete the following information):**
 - OS: Mac 14.6.1 (23G93)
 - Browser I tried in safari Version 17.6 (19618.3.11.11.5) and chrome Version 129.0.6668.71 (Official Build) (arm64)
 - and anyway my web host could reproduce the issue at their end on whatever browser they were using.

**Server
 - OS: I don't know this.
 - Version of Chamilo:  1.11.26
 - Version of PHP: 7.4

**Comment by riesaau** [comment]
More server info:-
Server
 - OS: [e.g. Linux Ubuntu 18.04/Windows 10]
 - Version of Chamilo:  1.11.26
 - Version of PHP: [e.g. 5.6, 7.2, ...]

**Comment by riesaau** [comment]
more server info:-
http://screens.kgix.net/hm/firefox_3hoa8aW4II.png

**Comment by ywarnier** [comment]
I don't think that's a Chamilo issue. I'll transfer this to discussions until we have a better understanding of the context.

**Reply by riesaau** [reply]
Thanks Yannick. I wonder if it is some php setting of the host that times out or something during upload, although the upload is fairly quick.

**Reply by ywarnier** [reply]
I feel like it is a possibility that there might be limits at the web server level (it doesn't look like PHP, and if it was you would probably get a clear error log line in your web server log).
In your first screenshot, you have a 403 Forbidden error. This looks to me like it could be linked to a limit at the web server level or, if there is a proxy or anything between you and the server (some hosting providers put that in the mix), this might be where it gets limited. This can also be a limit related to speed: uploading 100MB in 5s is not always the same as uploading 100MB in 60s, limits-wise...

I would file a ticket with your hosting provider, but not before looking into the details of the 403 error you are receiving. If you don't see any detail, then maybe mention the URL (full line containing bigupload.php) and the fact you get a 403. They can probably see more details in their logs.
Or maybe you have access to your error log and can look for any reference to "403" to get more details.

**Reply by ywarnier** [reply]
And if your internet connexion is really fast, maybe you can try to slow it down (send it through your mobile phone on 3G/4G (not 5G) for example - although this might have a cost :grin: )

**Comment by riesaau** [comment]
I'll talk to the hosting team at Kualo again.
Thanks for the tips.
Kind regards,
Riesa
